_posts/developers/apps/inrupt-tutorial/2020-11-06-00_getting-started.md

```diff
@@ -12,92 +12,88 @@ redirect_from:
 
 # Getting Started
 
+- [Prerequisites](#prerequisites)
+- [Build the Application](#build-the-application)
+- [Run the Application](#run-the-application)
+
```

Comment:
Since the page is so long, thought it might help to have a table of contents link to the sections on the page.